scripts/vic_utils

```diff
@@ -59,37 +61,202 @@ def main():
+                                    default=False)
+    grid_params_parser.add_argument("--vegparam_albedo",
+                                    type=str,
+                                    help="Flag indicating whether veg "
+                                         "parameter file contains albedo "
+                                         "values (default=False)",
+                                    default=False)
+    grid_params_parser.add_argument("--lai_src",
+                                    type=str,
+                                    help="Where to take lai values from; "
+                                         "can be one of: \'FROM_VEGLIB\', "
+                                         "\'FROM_VEGPARAM\' "
+                                         "(default=\'FROM_VEGLIB\')",
+                                    choices=['FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_VEGLIB")
+    grid_params_parser.add_argument("--fcan_src",
+                                    type=str,
+                                    help="Where to take fcanopy values from; "
+                                         "can be one of: \'FROM_DEFAULT\', "
+                                         "\'FROM_VEGLIB\', \'FROM_VEGPARAM\'; "
+                                         "\'FROM_DEFAULT\' = set them to 1.0; "
+                                         "(default=\'FROM_DEFAULT\')",
+                                    choices=['FROM_DEFAULT',
+                                             'FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_DEFAULT")
+    grid_params_parser.add_argument("--alb_src",
+                                    type=str,
+                                    help="Where to take albedo values from; "
+                                         "can be one of: \'FROM_VEGLIB\', "
+                                         "\'FROM_VEGPARAM\' "
+                                         "(default=\'FROM_VEGLIB\')",
+                                    choices=['FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_VEGLIB")
+    grid_params_parser.add_argument("--lake_profile",
+                                    type=str,
+                                    help="Flag indicating whether lake "
+                                         "parameter file contains lake "
+                                         "depth-area relationship "
+                                         "(default=False)",
+                                    default=False)
+    grid_params_parser.add_argument("--carbon",
+                                    type=str,
+                                    help="Flag indicating whether state "
+                                         "file contains carbon-cycle "
+                                         "variables (default=False)",
+                                    default=False)
```

Comment:
I see some instances in `ncparam2ascii_parser` in which the default value of a `str` option is also `False`.  I won't fix those, but I'm calling your attention to them in case you want to open an issue about it.